.circleci/config.yml

```diff
@@ -179,7 +257,7 @@ workflows:
 #            - hmpps-common-vars
 #          <<: *slack-fail-post-step
 
-      - e2e_local_test:
+      - integration_tests:
```

Comment:
Set the pipeline to run the new version of the test step, keeping the previous version until we are stable and happy to drop it.